renovate.json

```diff
@@ -2,33 +2,32 @@
   "timezone": "America/New_York",
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base"
+    "config:base",
+    "group:all"
```

Comment:
yeah that `group:all` seems unnecessary. I ended up with the following grouping in nova:

```
"packageRules": [
    {
      "matchPackageNames": ["github.com/openstack-k8s-operators/nova-operator/api"],
      "enabled": false
    },
    {
      "groupName": "openstack-k8s-operators",
      "matchPackagePrefixes": ["github.com/openstack-k8s-operators"],
      "excludePackageNames": ["github.com/openstack-k8s-operators/nova-operator/api"],
      "schedule": [
        "every weekend"
      ]
    },
    {
      "groupName": "k8s.io",
      "matchPackagePrefixes": [
        "k8s.io",
        "sigs.k8s.io"
      ],
      "schedule": [
        "every weekend"
      ],
      "allowedVersions": "< 1.0.0"
    }
  ]
```
* The first rule disables updating nova/api in nova
* The second rule creates a group for all the openstack-k8s-operators dependency. (excluding nova/api again)
* The third rule groups the k8s.io deps and pin them < 1.0.0 (see the versioning rule in their repo readme, it is changed)
This means the rest will be independent bumps but for nova that is only gomega and ginkgo which we do want to handle independently anyhow.

Reply:
hmmm, pretty sure I tried without 'group: all' and it left me with lots of PR's for each operator.